cmd/pj-rehearse/main.go

```diff
@@ -109,6 +105,14 @@ func main() {
 
 	prFields := logrus.Fields{"org": jobSpec.Refs.Org, "repo": jobSpec.Refs.Repo, "PR": getPrNumber(jobSpec)}
 	logger := logrus.WithFields(prFields)
+
+	if jobSpec.Type != "presubmit" {
```

Comment:
That should be a typed constant to import

Reply:
Addressed in https://github.com/openshift/ci-operator-prowgen/pull/49/commits/c8b379bfea9953128bf5db1957e6822b5091829e